client/src/redux/modal/modalActions.js

```diff
@@ -4,11 +4,12 @@ export const setModalVisibility = (
   modalType,
   modalVisibility,
   data,
+  onModalCloseCallback,
```

Comment:
I set up the `setModalVisibility` function to take an optional callback that will fire when the modal is closed. We'll use this so the show instance data refreshes on the dashboard after editing the instance. So, for example, if you change the show's name, now you'll see that change reflected immediately on the dashboard rather than having to reload the page.

 (@jberns  - fyi)